ethcore/res/ethereum/yolov1.json

```diff
@@ -0,0 +1,981 @@
+		"difficulty": "0x1",
+		"extraData": "0x00000000000000000000000000000000000000000000000000000000000000008a37866fd3627c9205a37c8685666f32ec07bb1b0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+		"gasLimit": "0x47b760",
+		"parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+		"seal": {
+			"ethereum": {
+				"nonce": "0x0000000000000000",
+				"mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+			}
+		},
+		"timestamp": "0x5ed754f1"
+	},
+	"nodes": [
+		"enode://9e1096aa59862a6f164994cb5cb16f5124d6c992cdbf4535ff7dea43ea1512afe5448dca9df1b7ab0726129603f1a3336b631e4d7a1a44c94daddd03241587f9@35.178.210.161:30303"
+	],
+	"accounts": {
+		"0x0000000000000000000000000000000000000000": {
+			"balance": "0x1"
+		},
+		"0x0000000000000000000000000000000000000001": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "ecrecover",
+				"pricing": {
+					"linear": {
+						"base": 3000,
+						"word": 0
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000002": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "sha256",
+				"pricing": {
+					"linear": {
+						"base": 60,
+						"word": 12
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000003": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "ripemd160",
+				"pricing": {
+					"linear": {
+						"base": 600,
+						"word": 120
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000004": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "identity",
+				"pricing": {
+					"linear": {
+						"base": 15,
+						"word": 3
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000005": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "modexp",
+				"pricing": {
+					"modexp": {
+						"divisor": 20
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000006": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "alt_bn128_add",
+				"pricing": {
+					"alt_bn128_const_operations": {
+						"price": 500
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000007": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "alt_bn128_mul",
+				"pricing": {
+					"alt_bn128_const_operations": {
+						"price": 6000
+					}
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000008": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "alt_bn128_pairing",
+				"alt_bn128_pairing": {
+					"base": 45000,
+					"pair": 34000
+				}
+			}
+		},
+		"0x0000000000000000000000000000000000000009": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "blake2_f",
+				"pricing": {
+					"blake2_f": {
+						"gas_per_round": 1
+					}
+				}
+			}
+		},
+		"000000000000000000000000000000000000000a": {
+			"balance": "0x1",
+			"builtin": {
+				"name": "bls12_381_g1_add",
+				"pricing": {
+					"bls12_const_operations_price": {
```

Comment:
After requested changes to 2537 integration and renames of structures you would need to fix fields in this spec (e.g. remove _price here).